app/models/guide_search.rb

```diff
@@ -0,0 +1,57 @@
+class GuideSearch
+  attr_reader :filter
+  attr_reader :order
+  attr_reader :query
+
+  def initialize
+    @filter = {}
+    @order = { _score: :desc }
+    @query = "*"
+  end
+
+  def self.search(query="*")
```

Comment:
Surrounding space missing in default value assignment.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.